apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -38,7 +40,16 @@ function SectionProgressSelector({
 
   const params = queryParams('view');
 
+  const v1Disabled = DCDO.get('disable-progress-v1', false);
+
   const displayV2 = React.useMemo(() => {
+    if (v1Disabled) {
```

Comment:
Am I reading this right that this also sets the user's default to be v2? So if we turned v1 off then needed to turn it on again, the user would need to switch back?